sample_tracking/app/import-samples.tsx

```diff
@@ -128,10 +142,12 @@ export default function ImportSamples() {
     }
 
     function handleDownloadClick() {
+        console.log("got here 1: " + errorSamples);
         if (errorSamples) {
+            console.log("got here 2");
             csvExporter.generateCsv(errorSamples);
```

Comment:
can generateCsv throw an error that we need to report to ths uer?

Reply:
According to the npm documentation it doesn't seem like it can throw an error but I added an alert if there are no error samples which would cause the CSV to not be downloaded.